### DISCUSSION TITLE
Use mesh deformation alone

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#1811] [Q&A]
Hello, I want to use SU2's mesh deformation module alone. For example, I manually provide a file specifying the deformation. I want to deform the mesh according to the file I provide. What should I do?

**Comment by bigfooted** [comment]
Easiest is to use FFD boxes, Have a look at the tutorial for species transport:
https://su2code.github.io/tutorials/Species_Transport/
In section 3, it is explained how to manually modify the Free Form Deformation box.

**Reply by Zcaic** [reply]
Thanks a lot！I'm going to have a look now。
Thank you！

**Reply by NAnand-TUD** [reply]
As a side note, if you want to move a mesh surface. You can also deform using a file: 
`DV_KIND= SURFACE_FILE  
DV_MARKER= (Blade) 
DV_FILENAME= MoveSurface.txt`

**Comment by islamuddin-iub** [comment]
ok
